web/src/search/helpers.tsx

```diff
@@ -74,6 +75,50 @@ export function toggleSearchFilter(query: string, searchFilter: string): string
     return (query.substring(0, idx).trim() + ' ' + query.substring(idx + searchFilter.length).trim()).trim()
 }
 
+export function getSearchTypeFromQuery(query: string): SearchType {
+    /** RegExp to match `type:$TYPE` in any part of a query. */
+    const getTypeName = /\btype:(?<type>\w*)\b/
+    const matches = query.match(getTypeName)
+
+    if (matches && matches.groups && matches.groups.type) {
+        /**
+         * In an edge case where multiple `type:` filters are used, if
+         * `type:symbol` is included, symbol results be returned, regardless of order,
+         * so we must check for `type:symbol`. For other types,
+         * the first `type` filter appearing in the query is applied.
+         **/
+        const getSymbolType = /\btype:symbol\b/
```

Comment:
```suggestion
        const symbolTypeRegex = /\btype:symbol\b/
```

`getSymbolType` sounds like a function name (same for `getTypeName` above)